Tutorial.md

```diff
@@ -11,477 +12,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
 > + [API Blueprint Map][map]
 > + [Tools for API Blueprint][Tooling Section]
 
-## Gist Fox API
-Without any further ado our *Gist Fox API* blueprint starts like this:
-
-    FORMAT: 1A
-
-    # Gist Fox API
-    Gist Fox API is a **pastes service** similar to [GitHub's Gist](http://gist.github.com).
-
-What have we just done? Let's break it line by line:
-
-#### Metadata
-
-    FORMAT: 1A
-
-Depending on the tools you will be using with your blueprint you might want to start your blueprint file with a metadata explicitly stating the API Blueprint version you want to use like in this case the version 1A.
-
-> **Note:** Should you need a clarification of some terms as used through this document refer to [API Blueprint Glossary of Terms][].
-
-#### API Name
-
-    # Gist Fox API
-
-Every good API should have a name. So does ours – *"Gist Fox API"*. The first Markdown header in our document represents just that – the API name.
-
-#### API Description
-
-    Gist Fox API is a **pastes service** similar to [GitHub's Gist][http://gist.github.com].
-
-The API Name might be followed by any arbitrary Markdown-formatted discussion. Preferably about your API.
-
-> **Note:** We will add much richer and more helpful discussion in the final version of this blueprint. Adding details about authentication, used media types and error handling to API description is always a great idea!
->
-> **Note:** For the sake of simplicity we will now omit user management and authentication. These features will be discussed in a future instalment of the API Blueprint Tutorial.
-
-## Markdown
-All you really need to write a blueprint is a text editor. A Markdown editor is even better. Anything you like. From Vi to Byword. Online editors are great too! You can even directly edit a blueprint on GitHub in the repository your service is living!
-
-If you are completely new to Markdown it is now the best time to learn about it. Just head over to [Markdown Tutorial][] and enjoy the ride. Just don't forget to come back we have some cool stuff too!
-
-Should you just need to refresh Markdown syntax you can always check the one and only [Gruber's Original][] documentation.
-
->  **Note:** It is not essentially required to know the Markdown syntax but it will help you a lot. The knowledge of Markdown will make some API Blueprint constructs more intuitive.
-
-## First Resource
-The first resource in our API is its root. The entry point to our API is defined like so:
+## API Blueprint
 
-    # Gist Fox API Root [/]
-    Gist Fox API entry point.
+The start of the Blueprint is going to look as follows:
```

Comment:
When you say "the start," do you mean the first iteration of the blueprint for this tutorial, or the beginning of the document?

> The start of the Blueprint looks as follows:

I would remove "is going to."

Also, are we capitalizing "Blueprint" when it is without the "API" on the front?